cfme/tests/intelligence/reports/test_views.py

```diff
@@ -46,6 +43,12 @@ def test_report_view(report, view_mode):
         caseimportance: high
         initialEstimate: 1/6h
     """
-    view = navigate_to(report, 'Details')
+    try:
+        view = navigate_to(report, "Details")
+    except CandidateNotFound:
+        report.browser.refresh()
+        view = navigate_to(report, "Details")
```

Comment:
@valaparthvi so this is full work around. @mshriver don't like `resetter` then Please add one Note Inline comment so in feature we can quick understand whats problem.